src/popup/components/TransactionOverview.vue

```diff
@@ -234,7 +237,12 @@ export default defineComponent({
       if (innerTx.value.function === TX_FUNCTIONS.claim) {
         transactionOwnerAddress = await decodeClaimTransactionAccount();
       }
-      ownershipAccount.value = getOwnershipAccount(transactionOwnerAddress);
```

Comment:
This change is a bit confusing for me. 
Why is `ownerAddress` not `transactionOwnerAddress`? And why is `ownershipAccount` not just whatever `getOwnershipAccount(transactionOwnerAddress)` returns? 
I suggest to add some comments here or to change the variable names to make this more clear